cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -66,6 +66,13 @@ class ReporterCommand : CliktCommand(
     ).convert { it.expandTilde() }
         .file(mustExist = true, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = true)
 
+    private val packageConfigurationFile by option(
+        "--package-configuration-file",
+        help = "The file containing the package configurations to read as input. This option does not have any effect" +
+                " in case a package configuration dir is specified."
```

Comment:
Ideally, this would be modeled by using `mutuallyExclusiveOptions` instead of this help text.

Reply:
Done.